### ISSUE TITLE
Create a list for Ublock origin.

### ISSUE THREAD

**Issue by STVGecko** [OhMyGuus/I-Still-Dont-Care-About-Cookies#1177]
Hello, can you create a list for ublock origin like the original "I don't care about cookie" extension ?

https://www.i-dont-care-about-cookies.eu/abp/

**Comment by RaptaG** [comment]
Yes, this would be huge! @OhMyGuus, could you please make a filter list? (Sorry for the ping, I just believe this issue is important)

**Comment by Mylloon** [comment]
Duplicate of #133

**Comment by krystian3w** [comment]
Duplicate was closed by wrong write bot?

https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/133#issuecomment-1427131152

> **We where not able to find the cookie banner**, it might already be fixed.
> If there is still a banner please resumbit and add a comment where it is thanks!

**Comment by azgaresncf** [comment]
I wanted to create it! ;)

**Comment by Cwpute** [comment]
I'd be glad if this was a thing yeah! ISDCAC is currently the best tool for refusing and hiding cookie banner, and it would be fantastic if its script was made available for opensource projects !
Since more and more browsers are integrating userscripts in their options, this would allow many Android users to benefit from it !

It would also kind of solve the discussion i had opened some time ago: https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/discussions/3831

**Comment by Cwpute** [comment]
would still be very much appreciated ! :)
